editor/plugins/script_editor_plugin.cpp

```diff
@@ -2044,63 +2115,9 @@ bool ScriptEditor::edit(const RES &p_resource, int p_line, int p_col, bool p_gra
 			p_resource->get_path().is_resource_file() &&
 			p_resource->get_class_name() != StringName("VisualScript") &&
```

Comment:
These two checks:
`p_resource->get_path().is_resource_file() && p_resource->get_class_name() != StringName("VisualScript")` 
ought be be moved into the new method `open_script_in_external_editor`.

Reply:
Agreed.